demo/interhand3d_img_demo.py

```diff
@@ -0,0 +1,246 @@
+    parser.add_argument(
+        '--show-ground-truth',
+        action='store_true',
+        help='If True, show ground truth keypoint if it is available.')
+    parser.add_argument(
+        '--show',
+        action='store_true',
+        default=False,
+        help='whether to show img')
+    parser.add_argument(
+        '--out-img-root',
+        type=str,
+        default=None,
+        help='Root of the output visualization images. '
+        'Default not saving the visualization images.')
+    parser.add_argument(
+        '--device', default='cuda:0', help='Device for inference')
+    parser.add_argument(
+        '--kpt-thr', type=float, default=0.3, help='Keypoint score threshold')
+
+    args = parser.parse_args()
+    assert args.show or (args.out_img_root != '')
+
+    coco = COCO(args.json_file)
+
+    # build the pose model from a config file and a checkpoint file
+    pose_model = init_pose_model(
+        args.pose_config, args.pose_checkpoint, device=args.device.lower())
+    dataset = pose_model.cfg.data['test']['type']
+
+    # load camera parameters
+    camera_params = None
+    if args.camera_param_file is not None:
+        camera_params = mmcv.load(args.camera_param_file)
+    # load ground truth joints parameters
+    gt_joint_params = None
+    if args.gt_joints_file is not None:
+        gt_joint_params = mmcv.load(args.gt_joints_file)
+
+    # load person bounding boxes
+    person_results_list = []
+    for image_id, image in coco.imgs.items():
+        image_name = osp.join(args.img_root, image['file_name'])
+
+        ann_ids = coco.getAnnIds(image_id)
+        person_results = []
+
+        capture_key = str(image['capture'])
+        camera_key = image['camera']
+        frame_idx = image['frame_idx']
+
+        for ann_id in ann_ids:
+            ann = coco.anns[ann_id]
+            if camera_params is not None:
+                camera_param = {
+                    key: camera_params[capture_key][key][camera_key]
+                    for key in camera_params[capture_key].keys()
+                }
+                camera_param = _transfer_interhand_camera_param(camera_param)
+            else:
+                camera_param = None
+            if gt_joint_params is not None:
+                joint_para = gt_joint_params[capture_key][str(frame_idx)]
+                gt_joint = np.concatenate([
+                    np.array(joint_para['world_coord']),
+                    np.array(joint_para['joint_valid'])
+                ],
+                                          axis=-1)
+            else:
+                gt_joint = None
+
+            det_result = {
+                'image_name': image_name,
+                'bbox': ann['bbox'],  # bbox format is 'xywh'
+                'camera_para': camera_param,
+                'keypoints_3d_gt': gt_joint
+            }
+            person_results.append(det_result)
+        person_results_list.append(person_results)
+
+    for i, person_results in enumerate(
```

Comment:
Are these person detection results or hand detection results?

Reply:
It's the detection results of hand. I have changed the name to det_results.